README.md

```diff
@@ -78,28 +91,16 @@ You should now be able to run superset using the `superset run` command:
 ```bash
 $ superset run -p 8088 --with-threads --reload --debugger
 ```
-However, OAuth login does not work yet as hq-superset needs a Postgres
-database created to store CommCare HQ data.
-
 
-### Create a Postgres Database Connection for storing HQ data
-
-- Create a Postgres database. e.g.
-  ```bash
-  $ createdb -h localhost -p 5432 -U commcarehq hq_data
-  ```
-- Log into Superset as the admin user created in the Superset
-  installation and initialization. Note that you will need to update
-  `AUTH_TYPE = AUTH_DB` to log in as admin user. `AUTH_TYPE` should be
-  otherwise set to `AUTH_OAUTH`.
```

Comment:
Just checking on why we removed this part?

Reply:
This [gets done automatically](https://github.com/dimagi/commcare-analytics/blob/d30cc792ecd236157356504bda5270fd9430a461/hq_superset/utils.py#L47) now by `get_hq_database()`.

Reply:
Oh, isn't this part about changing the auth_type to enable login via admin which is still applicable?

Reply:
Good point! We don't need to log in as admin to create the database any more, but it is useful to know how to log in as admin if there is any other reason to do so. I'll add that back.

Reply:
af4878f